iree/compiler/Api/BUILD

```diff
@@ -37,13 +37,13 @@ filegroup(
 
 filegroup(
     name = "python_compiler_api",
```

Comment:
Because these have (I think) some delicate touch points on the google3 side (they only exist for that). Figured we could update those when syncing and port back.